enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -196,8 +196,9 @@ export type MetabaseComponentTheme = {
   };
 
   /** Popover */
-  popover?: {
-    /** z-index of the popover. Useful for embedding components in a modal. Defaults to 4. */
+  popover: {
```

Comment:
(note to self: I might rename this to `overlays` or `floatingElements` in the user-facing theme options)

Reply:
For what it's worth, I think the terms do not have a universally agreed-upon meaning.

According to [this blogger](https://medium.com/design-bootcamp/popups-dialogs-tooltips-and-popovers-ux-patterns-2-939da7a1ddcd):

> A popover is a small overlay that appears close to a specific UI element, usually triggered by user interaction with it, such as hovering or clicking.

> An overlay is a broader term that describes various graphical elements or layers that are displayed on top of the existing user interface or content.

This is how I use the term "overlay" in this PR. That's partly inspired by the fact that we already put all of our Mantine floating elements in an "overlays" directory.

But there are other ways of using these words. According to [this post](https://www.nngroup.com/articles/popups/) from the Nielsen Norman group:

> A popup (also known as an overlay or popover) is a window or dialog that appears on top of the page content. 

And just to make things extra confusing, a Mantine [Overlay](https://mantine.dev/core/overlay/) is a translucent backdrop behind a modal.

One wrinkle is that I have never considered the left sidebar an overlay, even though it is quite similar to a sidesheet. The main difference, UX-wise, is that a sidesheet has a translucent backdrop.

Reply:
I've started a Design system glossary to help us avoid miscommunication caused by ambiguity: https://www.notion.so/metabase/Design-system-glossary-15a69354c9018069b206f4173aecc2b9